swatch-contracts/build.gradle

```diff
@@ -28,11 +18,11 @@ dependencies {
     implementation 'io.quarkus:quarkus-security'
     implementation 'io.quarkus:quarkus-smallrye-fault-tolerance'
     implementation 'io.quarkus:quarkus-smallrye-openapi'
-    implementation("io.quarkus:quarkus-liquibase")
-    implementation("io.quarkus:quarkus-jdbc-postgresql")
+    implementation 'io.quarkus:quarkus-jdbc-postgresql'
     implementation 'io.quarkus:quarkus-messaging-amqp'
     implementation 'io.quarkus:quarkus-messaging-kafka'
     implementation 'io.smallrye.reactive:smallrye-reactive-messaging-in-memory'
+    implementation 'org.apache.commons:commons-lang3'
```

Comment:
We should avoid adding dependencies with is not super justified. In this case, I think it's only for StringUtils.isEmpty, which we could create our own or use another alternatives within the Quarkus framework.

Reply:
I added this because it was already in use as a transitive dependency.  If you have an alternative that's already on the classpath, let me know and I'll use it.  Certainly we can write our own "isEmpty" method but do we put that in this project?  There's no common utility project used by the quarkus apps.  What happens when we need "isEmpty" in another Quarkus app?  Now we have two versions of the same function in two projects.  I don't really see the benefit to writing our own implementation unless we have a truly central place to put it.

Reply:
We were already using this dependency for some utils it provides:
![Screenshot From 2025-03-06 09-18-02](https://github.com/user-attachments/assets/981b7297-51ba-44c0-91b1-951362aba495)
So, let's use it here. We can always refactor this as part of another ticket.